aws-terraform/README.md

```diff
@@ -1,104 +1,113 @@
-# Deploying the 1Password SCIM Bridge in AWS cloud using Terraform 0.12.x
+# Deploying the 1Password SCIM Bridge in AWS/Terraform
 
-This example describes one of the simplest methods of deploying the 1Password SCIM bridge to AWS cloud.
+This document describes deploying the 1Password SCIM bridge to your Amazon Web Service cloud services using Terraform.
 
-## General Recommendations on deploying SCIM Bridge to public cloud or private infrastructure
 
-* SCIM requests must be secured via TLS using an API gateway or load balancer (transport security).
-* Anonymous access is not supported. A bearer token and session file is required for authentication with the SCIM Bridge and the 1Password service.
-* The session file is an encrypted file containing privileged 1Password account credentials. This file is created before the bridge application is deployed. The bearer token combined with the encrypted session file represent an authentication mechanism and must be treated extremely seriously in terms of security.
+## Preparing
 
-## Prepare your 1Password Account
+This is only an example of how you could deploy the 1Password SCIM Bridge to your existing AWS infrastructure. Utilizing the full suite of AWS/Terraform infrastructure is considered an advanced form of deployment, so feel free to modify it to suit your needs.
```

Comment:
> Utilizing the full suite of AWS/Terraform infrastructure is considered an advanced form of deployment, so feel free to modify it to suit your needs.

Minor quibble: I'm not a fan of this phrasing. Perhaps something like: "As an advanced form of deployment, the terraform deployment can be integrated into your existing infrastructure"

Up to you.